translations/bnpchs-final-2018-chairoikoguma.html

```diff
@@ -0,0 +1,107 @@
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Chairoikoguma (otherwise known as Koguma) is a bear that loves honey a lot. At one time in the honey forest, while wandering around peacefully, Koguma found the 
```

Comment:
```suggestion
  Chairoikoguma (otherwise known as Koguma) is a bear that loves honey very much. One time in the honey forest, while wandering around peacefully, Koguma found the 
```